hardware/MQTT.cpp

```diff
@@ -3160,14 +3198,29 @@ void MQTT::InsertUpdateSwitch(_tMQTTASensor* pSensor)
 			if (!root["color"]["b"].empty())
 				color_new.b = root["color"]["b"].asInt();
 			if (!root["color"]["c"].empty())
-				color_new.cw = root["color"]["c"].asInt();
+				color_new.cw = root["color"]["c"].asInt();		
```

Comment:
Could you undo this change, or remove the spaces at the end?

Reply:
Done